runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -415,14 +416,27 @@ jvmtiGetFrameLocation(jvmtiEnv* env,
 		rc = getVMThread(currentThread, thread, &targetThread, TRUE, TRUE);
 		if (rc == JVMTI_ERROR_NONE) {
 			J9StackWalkState walkState;
-
-			vm->internalVMFunctions->haltThreadForInspection(currentThread, targetThread);
-
-			walkState.walkThread = targetThread;
 			walkState.flags = J9_STACKWALK_INCLUDE_NATIVES | J9_STACKWALK_VISIBLE_ONLY | J9_STACKWALK_COUNT_SPECIFIED | J9_STACKWALK_RECORD_BYTECODE_PC_OFFSET;
 			walkState.skipCount = (UDATA) depth;
 			walkState.maxFrames = 1;
-			vm->walkStackFrames(currentThread, &walkState);
+
+#if JAVA_SPEC_VERSION >= 19
+			if (NULL == targetThread) {
+				j9object_t threadObject = J9_JNI_UNWRAP_REFERENCE(thread);
+				if (IS_VIRTUAL_THREAD(currentThread, threadObject)) {
+					j9object_t contObject = (j9object_t)J9VMJAVALANGVIRTUALTHREAD_CONT(currentThread, threadObject);
+					vmFuncs->walkContinuationStackFrames(currentThread, contObject, &walkState);
+				} else {
+					Assert_JVMTI_unreachable();
+				}
+			} else
+#endif /* JAVA_SPEC_VERSION >= 19 */
+			{
+				vmFuncs->haltThreadForInspection(currentThread, targetThread);
+				walkState.walkThread = targetThread;
+				vm->walkStackFrames(currentThread, &walkState);
+				vmFuncs->resumeThreadForInspection(currentThread, targetThread);
+			}
 			if (walkState.framesWalked == 1) {
```

Comment:
```suggestion
			if (1 == walkState.framesWalked) {
```

Reply:
May I ask why do we want to do this?

Reply:
Ref coding standard: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-conditions

To catch errors through the compiler if the equality is accidentally omitted:
- `if (1 = walkState.framesWalked) {`: compiler will throw an error.
- `if (walkState.framesWalked = 1) {`: compiler will consider this valid.